### PR TITLE
add airbyte swag

### LINKED ISSUE #1093: Add a swag opportunity from airbyte this hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Airbyte

Q. Where have you seen the announcement (URL)?  
A. : Here it is: https://airbyte.com/blog/announcing-airbytes-tentaculous-hacktoberfest-2023-edition

Q. What kind of swag (clothing, stickers, ...)?  
A. : Clothing

Q. Have you received swag from this?  
A. : No

Q. Does it ship worldwide?  
A. : Yes

Build or improve upon the Airbyte connectors to win cash prizes up to USD **4000**. You could also write quickstart guides to win cash prize up to **USD** **3000**. All contributions will be rewarded with credits which can be redemeed in the Airbyte Swag store for free swag.

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by aashishpanthi**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

I've added a airbyte swag opportunity. This fixes #1093

### CODE REVIEW

**Comment by aslafy-z** [comment]
Your branch has conflicts, please fix them 🙏

**Comment by aashishpanthi** [comment]
I've solved the merge conflict

File: data.json [hunk 1]
```diff
@@ -18,6 +18,14 @@
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
   {
+    "name": "Airbyte",
+    "difficulty": "medium",
+    "description": "Build or improve upon the Airbyte connectors to win cash prizes up to USD 4000. You could also write quickstart guides to win cash prize up to USD 3000. All contributions will be rewarded with credits which can be redemeed in the Airbyte Swag store for free swag.",
+    "reference": "https://airbyte.com/blog/announcing-airbytes-tentaculous-hacktoberfest-2023-edition",
+    "image": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRkRoqkXtCv9s_kq_9XBSBq5W8WAiVOoJinl65e9A1EoveKqroANhAeW0va79uKzwSl0MM&usqp=CAU",
```
Comment (aslafy-z) [inline data.json:25]:
```suggestion
    "image": "https://i.imgur.com/lCirQFS.jpg",
```

**Review by aslafy-z — CHANGES_REQUESTED** [review]

File: data.json [hunk 2]
```diff
@@ -18,6 +18,14 @@
...
   },
   {
+    "name": "Airbyte",
+    "difficulty": "medium",
+    "description": "Build or improve upon the Airbyte connectors to win cash prizes up to USD 4000. You could also write quickstart guides to win cash prize up to USD 3000. All contributions will be rewarded with credits which can be redemeed in the Airbyte Swag store for free swag.",
+    "reference": "https://airbyte.com/blog/announcing-airbytes-tentaculous-hacktoberfest-2023-edition",
+    "image": "https://i.imgur.com/lCirQFS.jpg",
+    "dateAdded": "2023-10-15T15:29:00.000Z",
+    "tags": ["clothing", "hacktoberfest"]
+},{
```
Comment (aslafy-z) [inline data.json:28]:
```suggestion
  },
  {
```

**Review by aslafy-z — APPROVED** [review]

**Pull request merged**